test/msw-api/setup-server/scenarios/cookies-request.test.ts

```diff
@@ -1,10 +1,14 @@
-import * as https from 'https'
+/**
+ * @jest-environment node
+ */
+import https from 'https'
```

Comment:
## Note

Do not use wild import. Do use `allowSyntheticDefaultImport` in TypeScript. This makes `@swc/jest` happy. Otherwise, module patching from the interceptors applies patches to a different export methods than the test gets.